packages/npm/@amazeelabs/recipes/recipes/add-storybook.ts.md

```diff
@@ -0,0 +1,215 @@
+# Add storybook UI and Tailwind
+
+```typescript
+// Check node version.
+$$('node -v', {
+  stdout: $$.minimalVersion('12'),
+});
+
+// Check yarn version.
+$$('yarn -v', {
+  stdout: $$.minimalVersion('1.0'),
+});
+```
+
+## Project setup
+
+Create a folders structure to contain the UI
+
+```typescript
+// Choose a project name.
+const { projectName } = $$.prompts({
+  type: 'text',
+  name: 'projectName',
+  message: 'What is the name of the project?',
+  validate: (name) =>
+    !/^[a-z][a-z\d_]+$/.test(name)
+      ? 'Project names must start with a letter and contain lower case letters, numbers and underscores only.'
+      : true,
+});
+
+$$(`mkdir -p packages/@${projectName}/ui`);
+$$.chdir(`packages/@${projectName}/ui`);
+```
+
+Initialize the Ui project folder with yarn
+
+```typescript
+// yarn init
+$$('yarn init -w -y');
```

Comment:
I think we have to add the project namespace, or else we can't reference the package properly with `import '@project/ui';`. And it has to be private, or lerna will try to publish it to npm (and fail because it can't publish to the `@project` namespace).

What does `-w` do?

```suggestion
$$('yarn init -w -y -p');
$$.file('package.json', (json) => ({
  ...json,
  name: `@${projectName}/ui`,
  description: `Design system for ${projectName.toUpperCase()}`,
});
```